docs/pages/ktextbox.vue

```diff
@@ -2,7 +2,74 @@
 
   <DocsPageTemplate apiDocs>
     <DocsPageSection title="Overview" anchor="#overview">
-      For design guidance, see the page on <DocsInternalLink href="/textfields" text="text fields" />.
+      For detailed design guidance, refer to the page on <DocsInternalLink href="/textfields" text="text fields" />.
+    </DocsPageSection>
+
+    <DocsPageSection title="Input Types and Features">
```

Comment:
Could you please update to

```suggestion
    <DocsPageSection title="Usage">
```

This uses language from other documentation pages. Whenever possible we'd want to re-use section titles, it will help with overall consistency.